cypress/support/fragments/circulation-log/searchPane.js

```diff
@@ -12,6 +13,17 @@ export default {
     ]);
   },
 
+  getLastWeekSearchParam() {
```

Comment:
may be these methods should be moved to dateTools as common methods

Reply:
I thought about that, but this method returns value with custom formmating. I didn't see were this formating need except inventory search

Reply:
let's move it into method related with circ log search  in dateTools. File already has a few special date frmatters

Reply:
Do you think it is good practice to collect special formatter inside general tools that ca be using in whole project?
May be better to use special formatter inside place where it need?